src/app/common/TagList.vue

```diff
@@ -27,9 +33,12 @@ interface LabelValueWithRoute extends LabelValue {
   route: RouteLocationNamedRaw | undefined
 }
 
-const props = defineProps<{
+const props = withDefaults(defineProps<{
   tags: LabelValue[] | Record<string, string> | null | undefined
-}>()
+  shouldTruncate?: boolean
+}>(), {
+  shouldTruncate: false,
```

Comment:
I think there was a note a while ago saying everyone should generally prefer using shorter names for things (so just `something` instead of `isSomething`, `hasSomething`, `shouldSomething`) 

It would be nice for us to try and stick to that where possible. So that being said, could we call this `truncate`?

Reply:
I’m not a fan of that decision because it makes for generally more weird-to-read code and sometimes straight up confusing code (most commonly when the variables are "simple" verbs or nouns like "copy", "truncate", "error", etc.; less so for things like "loading").

That being said, at least in a typed code base, one can usually discover that a variable is a boolean and that a prop is expected to get a boolean value.

I’m not sure that’s a rule we should/must follow and we should probably decide on that separately. For the time being, seeing as pretty much all our booleans are named in this is/has/are/should style, I’m gonna stick to this. It’s relatively easy to find cases of this later on and change them, if we want to.

Reply:
> As part of the Kongponents redesign effort, we’re going to attempt to standardize our component prop names. We’d like to move away from prefixing props, especially booleans, in favor of naming them without the prefix.

Would be kinda nice to start making our naming similar to the majority of our components, especially seeing as we've literally just upgraded to v9 so this standardization felt like a good time to start but 🤷 

> That being said, at least in a typed code base, one can usually discover that a variable is a boolean and that a prop is expected to get a boolean value.

Yeah completely agree with that one

Gimme 2 tics let me approve this